client/src/components/DisplayTracks/DisplayTracks.css

```diff
@@ -1,3 +1,26 @@
+article.background-image {
```

Comment:
I think a new file .css would have been better than use DisplayTracks.css.

Reply:
We will see with Yoann if it is a good idea or not